### ISSUE TITLE
index out of bounds when reform year different from start year. 

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/PolicyBrain#348]
This reform:

```
{ "2019": {
    "_AMT_trt1": [0.0],
    "_AMT_trt2": [0.0]
}
}
```

Returns this error message:

```
Sorry! Your calculation failed. Please re-enter parameters and try again :(.
Error: index 14 is out of bounds for axis 0 with size 14
```

The same reform for 2016 rather than 2019 is fine, so I think the problem is reforms that take place in years after the start year. In TaxBrain, we accomodate those with the `*,*,*,0` syntax

**Comment by hdoupe** [comment]
This issue was resolved. I cannot point to the exact PR that solved this. However, you can see this run which uses a reform with parameters first taking effect in 2017 and start year 2013:
https://www.ospc.org/taxbrain/30721/

**Comment by hdoupe** [comment]
Closed #348
